### PR TITLE
Handle empty string in work time edit text

### PR DESCRIPTION
**Pull request by wclausen**

### CODE REVIEW

**Comment by wclausen** [comment]
Resolves #1

File: app/src/test/java/com/cronus/zdone/TestTasksRepo.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ class TestTasksRepo : TasksRepository {
     }
 
     override fun getTimeData(): Observable<TimeProgress> {
-        return Observable.just(TimeProgress(15, 85))
+        return Observable.just(TimeProgress(15, 85, 9999))
```
Comment (z1lc) [inline app/src/test/java/com/cronus/zdone/TestTasksRepo.kt:21]: is this the default? jfyi the default on server is `120`

**Review by z1lc — APPROVED** [review]

File: app/src/test/java/com/cronus/zdone/TestTasksRepo.kt (hunk 1 above)
Comment (wclausen) [reply app/src/test/java/com/cronus/zdone/TestTasksRepo.kt:21]: just a rando number for tests

Comment (wclausen) [reply app/src/test/java/com/cronus/zdone/TestTasksRepo.kt:21]: whoever updated that class to include the new field forgot to update the tests

**Pull request merged**